views/js/qtiCreator/editor/gridEditor/droppable.js

```diff
@@ -220,9 +237,12 @@ define([
         //bind all event handlers:
         $el.on('mouseenter.gridEdit.gridDragDrop', '[class^="col-"]:not(.new-col), [class*=" col-"]:not(.new-col)', _.debounce(function(e){
```

Comment:
Just a suggestion... 
This line is too long.
To improve readability you can  put `_.debounce(function(e){` on a new line

Reply:
Developers are still using the dash to defined that variable is used privately.